lib/command/diff.php

```diff
@@ -25,7 +25,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $io->title('YDeploy diff');
 
-        $tables = rex_sql::showTables(1, rex::getTablePrefix());
+        $sql = rex_sql::factory();
+        $tables = $sql->getTables(1, rex::getTablePrefix());
```

Comment:
Der DBID-Param existiert nicht in der Methode. Der wird bereits bei factory gesetzt.